src/main/java/TestFile.java

```diff
@@ -32,4 +32,11 @@ private static void onMoreVariationOfIncorrectComparingToItself() {
             // whatever
         }
     }
+
+    private static void inappropriateIndentation() {
+        int x = 5;
+        if (x > 0) {
+            // dummy code
+        }
+    }
```

Comment:
[Checkstyle] WARNING: 'method def rcurly' have incorrect indentation level 4, expected level should be 2.